rpcs3/Emu/System.cpp

```diff
@@ -1001,8 +1003,11 @@ game_boot_result Emulator::Load(const std::string& title_id, bool add_only, bool
 
 						if (obj == elf_error::ok)
 						{
+							std::unique_lock lock(sprx_mtx);
+
 							if (auto prx = ppu_load_prx(obj, path))
```

Comment:
```suggestion
							if (auto prx = (std::scoped_lock(sprx_mtx), ppu_load_prx(obj, path)))
```

Reply:
Scoped lock is for locking 2 or more mutexes at the same time, in general.

Reply:
It's also a bit longer to type. I recommend using lock_guard in most cases as usual.